### PR TITLE
ADDED security check

### PR DESCRIPTION
**Pull request by liamsorsby**
This should help show security vulnerabilities before they become and issue. Addresses #797

### CODE REVIEW

**Comment by liamsorsby** [comment]
@makasim Unfortunately, I tried to fix the tests by updating the version of Doctrine ORM. However, version 2.6.3 fixes the 'continue 2' regression in PHP 7.3 but they have also removed support for PHP 7.0 😞

**Comment by makasim** [comment]
We could remove support of 7.0
No problem

File: bin/release [hunk 1]
```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env bash
-
-# TODO: prepare changelog
```
Comment (makasim) [inline bin/release]: Why did you remove the file?

Comment (liamsorsby) [reply bin/release]: @makasim I just ran `git rm -rm . —cached && git add .`
It seems that the file matches one of the gitignore patterns
I can fix the gitignore though if it’s required.

**Comment by liamsorsby** [comment]
I’ll amend the PR to fix the gitignore and remove support for PHP 7.0 as long as your happy with it as security support has already ended.

File: bin/release (hunk 1 above)
Comment (liamsorsby) [reply bin/release]: I've reverted the deletion of this file

**Comment by pierredup** [comment]
@liamsorsby `sensiolabs/security-checker` is abandoned and replaced by https://github.com/fabpot/local-php-security-checker. Can you please update the PR to use the new binary?

**Comment by pierredup** [comment]
Closing in favour of #904.
